types/node/test/buffer.ts

```diff
@@ -397,7 +397,5 @@ buff.writeDoubleBE(123.123);
 buff.writeDoubleBE(123.123, 0);
 
 {
-    // The 'as any' is to make sure the Global DOM Blob does not clash with the
-    //  local "Blob" which comes with node.
-    resolveObjectURL(URL.createObjectURL(new Blob(['']) as any)); // $ExpectType Blob | undefined
+    resolveObjectURL("hello"); // $ExpectType Blob | undefined
```

Comment:
I suspect the original test snippet came from documentation somewhere. But the only type `resolveObjectURL()` accepts is a `string`, so I replaced with an actually string instead of referencing `URL` (which doesn't exist in this package)